### PR TITLE
chore: nil check when creating stressor

### LINKED ISSUE #2654: Chaos Dashboard panic when creating new chaos experiment
## Bug Report

**What version of Kubernetes are you using?**

**What version of Chaos Mesh are you using?**

**What did you do?**

**What did you expect to see?**

**What did you see instead?**  

27641639493373_ pic_hd

**Output of chaosctl**

### PR DESCRIPTION
**Pull request by STRRL**
Signed-off-by: STRRL <str_ruiling@outlook.com>

### What problem does this PR solve?

close #2654

### Related changes

- [ ] Need to update `chaos-mesh/website`
- [ ] Need to update `Dashboard UI`
- Need to **cheery-pick to release branches**
  - [ ] release-2.1
  - [ ] release-2.0

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] No code
- [x] Manual test (add steps below)

Side effects

- [ ] Breaking backward compatibility

### Release note 

```text
Please add a release note.

You can safely ignore this section if you don't think this PR needs a release note.
```

### DCO

If you find the DCO check fails, please run commands like below (Depends on the actual situations. For example, if the failed commit isn't the most recent) to fix it:

```shell
git commit --amend --signoff
git push --force
```

### CODE REVIEW

**Comment by STRRL** [comment]
PTAL

/cc @chaos-mesh/committers @chaos-mesh/maintainers

File: pkg/apiserver/experiment/experiment.go [hunk 1]
```diff
@@ -346,16 +346,25 @@ func (s *Service) createStressChaos(exp *core.ExperimentInfo, kubeCli client.Cli
...
+	} else if originStressors.CPUStressor != nil && originStressors.CPUStressor.Workers > 0 &&
+		(originStressors.MemoryStressor == nil || originStressors.MemoryStressor.Workers <= 0) {
 		stressors = &v1alpha1.Stressors{
-			CPUStressor: exp.Target.StressChaos.Stressors.CPUStressor,
+			CPUStressor: originStressors.CPUStressor,
 		}
 	} else {
-		stressors = exp.Target.StressChaos.Stressors
+		stressors = originStressors
 	}
```
Comment (WangXiangUSTC) [inline pkg/apiserver/experiment/experiment.go:368]: Are we allowed to have both `CPUStressor.Workers > 0` and `MemoryStressor.Workers > 0`?

Comment (STRRL) [reply pkg/apiserver/experiment/experiment.go:368]: oops, I think it's valid for setting `workers` for both CPU and memory. I would change that.

**Review by g1eny0ung — APPROVED** [review]
LGTM

**Review by WangXiangUSTC — APPROVED** [review]
LGTM

**Comment by STRRL** [comment]
/run-e2e-tests

:(

**Pull request merged**